docs/tools/terraform/reference/cookbook/kafka-debezium-postgres-source.rst

```diff
@@ -0,0 +1,197 @@
+Apache Kafka® with Debezium source connector for PostgreSQL®
+==========================================================
+
+The `Aiven Terraform Provider <https://registry.terraform.io/providers/aiven/aiven/latest/docs>`_ is a great choice for provisioning an Aiven for Apache Kafka® cluster with Kafka Connect enabled and the `Debezium source connector for PostgreSQL® <https://developer.aiven.io/docs/products/kafka/kafka-connect/howto/debezium-source-connector-pg.html>`_ configured.
+
+Let's check out the following diagram to understand the setup.
+
+.. mermaid::
+
+   flowchart LR
+      id1[(PostgreSQL Database)]
+      subgraph Kafka Connect
+      Debezium-Source-Connector-PG
+      end
+      subgraph Apache Kafka
+      Topic
+      end
+      id1 --->|wal2json plugin| Debezium-Source-Connector-PG -->|publishes changes| Topic
+
+Describe the setup
+------------------
+
+This terraform recipe will provision one Aiven PostgreSQL database service, one Aiven for Apache Kafka service, a separate Aiven for Apache Kafka Connect 
+service with a Debezium source connector for PostgreSQL enabled and configured to connect to the PostgreSQL database and capture any change in tables.
+As soon as any of the monitored tables is inserted or updated with new data, the Debezium connector will capture the data change and convert table data into
+json payload and produce messages to the relevant Kafka topic. 
+Some of these services are created in one cloud provider and some in another cloud provider, to demonstrate how easy it is with Aiven to integrate services across 
+multiple cloud vendors.
+
+.. Warning::
+
+    Creating a `dedicated Aiven for Apache Kafka Connect service <https://developer.aiven.io/docs/products/kafka/kafka-connect/getting-started.html#apache-kafka-connect-dedicated-cluster>`_ is Aiven's suggested option. Having Kafka Connect running on the same nodes as Apache Kafka increases the load on the nodes possibly making the cluster more unstable. 
```

Comment:
Be careful with wording here. Can having Kafka Connect on the same nodes always make the cluster more unstable? Is this Aiven's recommended option for all scenarios? 

> Aiven provides the option to run Kafka Connect on the same nodes as your Kafka cluster, sharing the resources. This is a low-cost way to get started with Kafka Connect but it is not recommended for Production environments, or scenarios where the deployed Connect cluster needs to be in a different cloud provider/region to the Kafka cluster. For these situations, Aiven recommends the use of a dedicated Aiven for Apache Kafka Connect service